cnb_image.go

```diff
@@ -496,24 +608,24 @@ func (i *CNBImageCore) SetCreatedAtAndHistory() error {
 	return err
 }
 
-func getConfigFile(image v1.Image) (*v1.ConfigFile, error) {
+func GetConfigFile(image v1.Image) (*v1.ConfigFile, error) {
```

Comment:
Why are we making the method Public?

Reply:
reverting back changes :)

Reply:
I wanted to make some changes in `local` and `remote` packages using these functions to avoid duplication, but now there is no need of these to be public, making these functions to private